rotkehlchen/tasks/calendar.py

```diff
@@ -410,6 +422,57 @@ def maybe_create_airdrop_claim_reminder(self) -> None:
             error_msg='Failed to add the airdrop claim reminders',
         )
 
+    def maybe_create_l2_bridging_reminder(self) -> None:
+        """Creates calendar reminders for L2 bridge claims (7 days after deposit).
+        Tracks deposits on Base, Optimism, and Arbitrum networks.
+        """
+        bridge_events: list[EvmEvent] = []
+        locations_to_counterparties = (
+            (Location.BASE, CPT_BASE),
+            (Location.OPTIMISM, CPT_OPTIMISM),
+            (Location.ARBITRUM_ONE, CPT_ARBITRUM_ONE),
+        )
+        db_history_events = DBHistoryEvents(database=self.database)
+        with self.database.conn.read_ctx() as cursor:
+            for location, counterparty in locations_to_counterparties:
+                bridge_events.extend(db_history_events.get_history_events(
+                    cursor=cursor,
+                    has_premium=True,
+                    filter_query=EvmEventFilterQuery.make(
+                        and_op=True,
+                        location=location,
+                        counterparties=[counterparty],
+                        event_types=[HistoryEventType.DEPOSIT],
+                        event_subtypes=[HistoryEventSubType.BRIDGE],
+                    ),
+                ))
+
+        now = ts_now_in_ms()
+        bridge_calendar_entries: list[int] = []
+        for bridge_event in bridge_events:
+            if now - bridge_event.timestamp > WEEK_IN_SECONDS * 1000:
+                continue
+
+            try:
+                if (entry_id := self.create_or_update_calendar_entry_from_event(
+                        name=f'Claim {bridge_event.balance.amount} {bridge_event.asset.resolve_to_asset_with_symbol().symbol} bridge deposit on Ethereum',  # noqa: E501
+                        event=bridge_event,
+                        color=BRIDGE_CALENDAR_COLOR,
+                        counterparty=bridge_event.counterparty,  # type: ignore[arg-type]  # counterparty is always present
+                        timestamp=ts_ms_to_sec(TimestampMS(bridge_event.timestamp + WEEK_IN_SECONDS)),  # noqa: E501
+                        description=f'Bridge deposit of {bridge_event.balance.amount} {bridge_event.asset.resolve_to_asset_with_symbol().symbol} is ready to claim on Ethereum',  # noqa: E501
+                )) is not None:
+                    bridge_calendar_entries.append(entry_id)
+            except UnknownAsset:
+                log.exception(f'Unable to add reminder for brige event with hash {bridge_event.identifier} on {bridge_event.location.name}')  # noqa: E501
```

Comment:
As I said in my last review. You need the tx hash. The DB identifier means nothing.

```suggestion
                log.exception(f'Unable to add reminder for bridge event with hash {bridge_event.tx_hash} on {bridge_event.location.name}')  # noqa: E501
```